backend/app/graphql/__init__.py

```diff
@@ -36,6 +42,23 @@ class RootMutation(
 def init_app(app):
     with app.app_context():
         # Add your services here: services["service_name"] = ...
+        services["user_service"] = UserService(logger=current_app.logger)
+        services["email_service"] = EmailService(
+            logger=current_app.logger,
+            credentials={
+                "refresh_token": os.getenv("MAILER_REFRESH_TOKEN"),
+                "token_uri": "https://oauth2.googleapis.com/token",
+                "client_id": os.getenv("MAILER_CLIENT_ID"),
+                "client_secret": os.getenv("MAILER_CLIENT_SECRET"),
+            },
+            sender_email=os.getenv("MAILER_USER"),
+            display_name="Display Name",
```

Comment:
Nit (optional): At some point we might want to change this to something more indicative of FCK.